doc/connectivity/networking/overview.rst

```diff
@@ -26,19 +26,18 @@ can be disabled if not needed.
     IPv6 prefix count can be configured at build time.
```

Comment:
Should this also mention IPv6 PE ([RFC 8981](https://datatracker.ietf.org/doc/html/rfc8981))?